tests/posit/connect/test_users.py

```diff
@@ -1,51 +1,25 @@
 import pandas as pd
+import pytest
 import responses
 
+from requests import HTTPError
+
 from posit.connect.client import Client
 
+from .api import load_mock
+
 
 class TestUsers:
     @responses.activate
-    def test_get_users(self):
+    def test_users_find(self):
         responses.get(
             "https://connect.example/__api__/v1/users",
             match=[
                 responses.matchers.query_param_matcher(
                     {"page_size": 2, "page_number": 1}
                 )
             ],
-            json={
-                "results": [
-                    {
-                        "email": "alice@connect.example",
-                        "username": "al",
-                        "first_name": "Alice",
-                        "last_name": "User",
-                        "user_role": "publisher",
-                        "created_time": "2017-08-08T15:24:32Z",
-                        "updated_time": "2023-03-02T20:25:06Z",
-                        "active_time": "2018-05-09T16:58:45Z",
-                        "confirmed": True,
-                        "locked": False,
-                        "guid": "a01792e3-2e67-402e-99af-be04a48da074",
-                    },
-                    {
-                        "email": "bob@connect.example",
-                        "username": "robert",
-                        "first_name": "Bob",
-                        "last_name": "Loblaw",
-                        "user_role": "publisher",
-                        "created_time": "2023-01-06T19:47:29Z",
-                        "updated_time": "2023-05-05T19:08:45Z",
-                        "active_time": "2023-05-05T20:29:11Z",
-                        "confirmed": True,
-                        "locked": False,
-                        "guid": "87c12c08-11cd-4de1-8da3-12a7579c4998",
-                    },
-                ],
-                "current_page": 1,
-                "total": 3,
-            },
+            json=load_mock("v1/users?page_number=1&page_size=2.json"),
```

Comment:
I've made the file paths match the queries in order to make it clear from the file system what they are, and I also aspire to simplifying/encapsulating some of this repeated `responses.get()` invocation further.